backend/.env.example

```
@@ -1,3 +1,3 @@
 GEMINI_API_KEY=your_gemini_api_key_here
 PORT=3011
-WS_PORT=8888
\ No newline at end of file
+WS_PORT=3002
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)